test/language/comments/hashbang-module.js

```diff
@@ -0,0 +1,10 @@
+#!
```

Comment:
add other test files verifying these characters can't be escaped? one for `#` and another for `!`

Reply:
Let’s try different types of escapes: https://github.com/v8/v8/blob/effb7ad728cf13da143fae41b03b0525f4ff65d5/test/cctest/test-parsing.cc#L11446-L11455

Reply:
cool! I'll work on some follow ups for this PR and this is a good material